mercury/templates/index.html

```diff
@@ -13,54 +13,49 @@
             <br>
-            Telemetry platform for monitoring <a href="https://www.nyutandonmotorsports.com/" target="_blank">NYU Tandon's Motorsports</a> SAE BAJA competition vehicle. This platform supports
+            Telemetry platform for monitoring <a href="https://www.nyutandonmotorsports.com/" target="_blank" style="color: var(--light-text); text-decoration: underline;">NYU Tandon's Motorsports</a> SAE BAJA competition vehicle. This platform supports
             reception and visualization of data from the competition vehicle along with several auxiliary features.
             <br>
             <br>
             <a href="https://nyu-mercury-grafana.herokuapp.com/d/BjN37GJZk/nyu-motorsports-telemetry-dashboard?orgId=1&refresh=5s" target="_blank">
-                <div class="tooltip homepage" id="grafana-text-box">
+                <div class="tooltip" id="grafana-text-box">
                     <h3>Dashboard</h3>
-                    Launches a data visualization tool in a new window.
-                    <span class="tooltiptext">This tool queries the team's database and uses the return values to populate several graphs. These graphs can be configured to fit the team's needs. Graphs can also be added or deleted as new data sources become available. </span>
+                    Launches a data visualization tool in a new window. This tool queries the team's database to populate several graphs. These graphs are fully configurable and can be added or deleted as new data sources become available. </span>
                 </div>
             </a>
             <br>
 
             <a href="{% url 'mercury:stopwatch' %}">
-                <div class="tooltip homepage" id="grafana-text-box">
+                <div class="tooltip" id="grafana-text-box">
                     <h3>Stopwatch</h3>
-                    Time the race with the Stop Watch feature.
-                    <span class="tooltiptext">This allows for overall time, as well as individual lap times, to be tracked in the user's browser.</span>
+                    Time the race with the Stopwatch feature. Track overall time and individual laps in the browser.
                 </div>
             </a>
             <br>
 
             <a href="{% url 'mercury:dashboard' %}">
-                <div class="tooltip homepage" id="grafana-text-box">
+                <div class="tooltip" id="grafana-text-box">
                     <h3>Raw Telemetry</h3>
-                    This portal allows the raw data to be viewed.
-                    <span class="tooltiptext">This is useful for debugging and understanding the models behind the data visualization portal. Data generated in the simulator will appear here, as well as any data from vehicle sensors.</span>
+                    This portal allows the raw data to be viewed. Useful for debugging and understanding the models behind the data visualization portal. Data generated in the simulator and/or vehicle sensors will appear here.
```

Comment:
"portal" seems like the wrong word to me. Maybe "feature", or "app"?

Reply:
I'm gonna reword the first sentence to say "Allows the raw data to be viewed"